src/main/java/io/airbrake/javabrake/NoticeBuilder.java

```diff
@@ -0,0 +1,74 @@
+package io.airbrake.javabrake;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class NoticeBuilder {
+  Map<String, Object> context;
+  Map<String, Object> params;
+  Map<String, Object> session;
+  Map<String, Object> environment;
+  Throwable exception;
+  List<NoticeError> errors;
```

Comment:
A reference to `Notifier` can be kept internally as a `final` field.

Reply:
I did not get it. Can you please explain?

Reply:
This would be needed to support the `send()` method mentioned above